google/cloud/pubsub_v1/types.py

```diff
@@ -37,6 +39,16 @@
 from google.pubsub_v1.types import pubsub as pubsub_gapic_types
 
 
+TimeoutType = Union[
+    None,
```

Comment:
Did we mean to add `None`?  Was this my fault?

This feels like a breaking change.  We didn't accept `None` before.